requirements.txt

```diff
@@ -1,11 +1,12 @@
-django>=2.1,<2.2
-https://github.com/opencivicdata/python-opencivicdata-django/archive/2.5.0.zip
-pupa==0.9.0
+django>=2.2,<2.3
+opencivicdata>=3.1.0
```

Comment:
Includes adding extras to BillDocument: https://github.com/opencivicdata/python-opencivicdata/commit/24ab713aeea50cbf6d88d747868ed9cdae7ea1f5.